docs/capabilities.md

```diff
@@ -240,6 +241,7 @@ CAP_ODBC_REBIND_SKIP_UNBIND | Set to 'yes' to force the Tableau native ODBC prot
 CAP_ODBC_SUPPORTS_LONG_DATA_BULK | Set to 'yes' if driver can fetch multiple long-data rows at a time. | no | no |
 CAP_ODBC_SUPPORTS_LONG_DATA_ORDERED | Set to 'yes' if driver requires long-data columns to come after non-long-data ones | no | no |
 CAP_ODBC_SUPPRESS_CATALOG_NAME | Set 'yes' to suppress passing catalog name for SQLTables, SQLPrimaryKeys, SQLForeignKeys, SQLStatistics calls | &ndash; | &ndash;
+CAP_ODBC_INCLUDE_CATALOG_NAME | Set 'yes' to include the catalog (database) name in fully qualified table names in queries, as Tableau does not explicitly set the catalog and it must be in the ODBC connection string. | yes | &ndash;
```

Comment:
Set 'yes' to include the catalog (database) name in the fully qualified table name